nbbuild/antsrc/org/netbeans/nbbuild/reporting/RatHtmlReportTask.java

```diff
@@ -0,0 +1,295 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.nbbuild.reporting;
```

Comment:
Does this really deserve a separate package? There is a lot of reporting tasks already present in `org.netbeans.nbbuild`... Moving into `org.netbeans.nbbuild` and merging `ModuleInfo` to be static inner class would, in my opinion, be more consistent.

Reply:
This sounds good for me